bin/bb8/status.py

```diff
@@ -35,7 +35,7 @@ def normalize_timestamp(string, timezone=None):
 
 
 def last_modified_remote(fm: RemoteFileManager, paths: RemotePaths):
-    remote_cmd = 'find -L {} -type f -print0'.format(paths.data) + \
+    remote_cmd = 'find -L {} -type f -o -type d -print0'.format(paths.data) + \
```

Comment:
When I run this command it only outputs directory timestamps, not directories and files...

Reply:
You're right. Gah, back to the drawing board on that command.

Reply:
So it seems like it does both files and directories when I leave off the `-print0` flag...

Reply:
It seems that moving `-print0` to before the matchers works. i.e:

```
find -L . -print0 -type f -o -type d
```

Reply:
Tested with new version and it tracks both files and directory changes.